### PR TITLE
7.38.0 changelog

### PR DESCRIPTION
**Pull request by kacper-murzyn**
7.38.0 changelog

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -2,6 +2,136 @@
...
+
+- Please refer to the `7.38.0 tag on integrations-core <https://github.com/DataDog/integrations-core/blob/master/AGENT_CHANGELOG.md#datadog-agent-version-7380>`_ for the list of changes on the Core Checks
+
+
+.. _Release Notes_7.38.0_New Features:
+
+New Features
+------------
+
+- Add `otlp_config.debug.loglevel` to determine log level when the OTLP Agent receives metrics/traces for debugging use cases.
```
Comment (mx-psi) [inline CHANGELOG.rst:25]: Let's move this to Enhancements

```suggestion
```

File: CHANGELOG.rst [hunk 2]
```diff
@@ -2,6 +2,136 @@
...
+- The Agent now embeds Python-3.8.13, an upgrade from
+  Python-3.8.11.
+
+- APM: Updated Rare Sampler default configuration values to sample traces more uniformly across environments and services.
+
+- Support Exponential Histograms with delta aggregation temporality.
+
+- The Windows installer now supports grouped Managed Service Accounts.
+
+- Enable https monitoring on arm64 with kernel >= 5.5.0.
```
Comment (mx-psi) [inline CHANGELOG.rst:84]:
```suggestion
- Enable https monitoring on arm64 with kernel >= 5.5.0.

- Add ``otlp_config.debug.loglevel`` to determine log level when the OTLP Agent receives metrics/traces for debugging use cases.
```

File: CHANGELOG.rst [hunk 3]
```diff
@@ -2,6 +2,136 @@
...
+- APM: Add support for PostgreSQL JSON operators in the SQL obfuscate package.
+
+- The OTLP ingest endpoint now supports the same settings and protocol as the OpenTelemetry Collector OTLP receiver v0.54.0 (OTLP v0.18.0).
+
+- The Agent now embeds Python-3.8.13, an upgrade from
+  Python-3.8.11.
+
+- APM: Updated Rare Sampler default configuration values to sample traces more uniformly across environments and services.
+
+- Support Exponential Histograms with delta aggregation temporality.
```
Comment (mx-psi) [inline CHANGELOG.rst:80]:
```suggestion
- The OTLP ingest endpoint now supports Exponential Histograms with delta aggregation temporality.
```

File: CHANGELOG.rst [hunk 4]
```diff
@@ -2,6 +2,136 @@
...
+
+
+.. _Release Notes_7.38.0_Deprecation Notes:
+
+Deprecation Notes
+-----------------
+
+- Use `instrumentation_scope_metadata_as_tags` instead of `instrumentation_library_metadata_as_tags`
+  in favor of https://github.com/open-telemetry/opentelemetry-proto/releases/tag/v0.15.0
+  Both must not be set at the same time.
```
Comment (mx-psi) [inline CHANGELOG.rst:94]:
```suggestion
- Deprecate``otlp_config.metrics.instrumentation_library_metadata_as_tags`` in 
  in favor of ``otlp_config.metrics.instrumentation_scope_metadata_as_tags``.
```

**Review by mx-psi — APPROVED** [review]

**Review by paulcacheux — APPROVED** [review]

**Review by ahmed-mez — APPROVED** [review]

**Pull request merged**